esti/lakectl_util.go

```diff
@@ -279,3 +281,26 @@ func normalizeAccessKeyID(output string) string {
 func normalizeSecretAccessKey(output string) string {
 	return reSecretAccessKey.ReplaceAllString(output, "secret_access_key: <SECRET_ACCESS_KEY>")
 }
+
+func getCommitter(t testing.TB) string {
```

Comment:
Can you please make `getCommitter` and `getAuthor` public, for being used in other locations?